src/test/Components/ImagesTable/ImagesTable.test.js

```diff
@@ -421,6 +421,7 @@ describe('Images Table', () => {
           <BrowserRouter>
             <ImageLink
               imageStatus={mockStatus[compose.id].image_status}
+              imageType={compose.request.image_requests[0].image_type}
```

Comment:
I was able to get the test to pass without this line, what's it for?

Reply:
The tests kept failing without the line, but you're right, they pass now. Seems like changes from PR #831 fixed the previous problem. 

But at the same time I'm newly getting a warning both with and without the line, as `imageName` is now marked as required. I'm not entirely sure why

Reply:
I'm planning on a huge refactor of the <ImagesTable /> component soon which will include hugely simplifying all of the child components (current plan is to only pass in `composeId` as the sole prop and then have the child component itself get everything it needs using Redux selectors). So I'm not super worried about this but it might be worth removing because it's confusing.

Let's do get the warning fixed though.

Reply:
The two ways to fix the warning I can see right now would be providing the tests with an empty strings for the required fields or removing the `isRequired` for the two props in `ImageLink.js` on lines 45 and 46. 

I'm leaning towards the empty strings in tests. Although it adds a bit of a clutter to the testing file the needed functionality of the code wouldn't be changed.

Reply:
Empty strings in the tests will be fine... this will all get cleaned up in the refactor I'm planning anyway (and this is a good example of why the refactor I want to do is needed).